packages/types/src/exchange/provider.ts

```diff
@@ -0,0 +1,5 @@
+export type ExchangeProvider = `${ExchangeProviders}`
```

Comment:
export const -> the enum to export as value at namespace lvl, check the ResolverNameServer example.